examples/select_file_with_information_panel/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "select_file_with_information_panel"
```

Comment:
Yes, agreed. Should I also rename the `InformationPanel` struct or leave it like this? We can also go back to `information_panel` feature, if you prefer that.

Reply:
Hard to say, but I think I prefer `information_view`.

`metadata_view` is not a good fit because we are displaying more information than just metadata (e.g. thumbnails).
`information_panel` is not a good fit since it can be used somewhere other than a panel